src/main/resources/liquibase/tenant/scripts/v3.5.3/alter-access-types-table.xml

```diff
@@ -0,0 +1,45 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+  <changeSet id="MODKBEKBJ-435@@drop-columns-from-access-types-table" author="psmagin">
+    <dropColumn tableName="access_types" columnName="created_by_username" />
+    <dropColumn tableName="access_types" columnName="created_by_last_name"/>
+    <dropColumn tableName="access_types" columnName="created_by_first_name"/>
+    <dropColumn tableName="access_types" columnName="created_by_middle_name"/>
+    <dropColumn tableName="access_types" columnName="updated_by_username"/>
+    <dropColumn tableName="access_types" columnName="updated_by_last_name"/>
+    <dropColumn tableName="access_types" columnName="updated_by_first_name"/>
+    <dropColumn tableName="access_types" columnName="updated_by_middle_name"/>
+  </changeSet>
+
+  <changeSet id="MODKBEKBJ-435@@add-foreign-keys-from-access-types-to-kb-users" author="psmagin">
+    <addForeignKeyConstraint
```

Comment:
it's better to have statements separately. thus they can be applied independently: if one fails it can be reapplied later because it belongs to different changeset
see https://www.liquibase.org/get-started/best-practices
![image](https://user-images.githubusercontent.com/42244720/87431663-a4be1e00-c5ef-11ea-876a-f6aaaf6a42ef.png)